.github/workflows/dependabot_update.yml

```diff
@@ -38,10 +38,12 @@ jobs:
         with:
           node-version: 20
 
-      - name: Lerna bootstrap
+      # Running npm ci also runs lerna bootstrap,
+      # which does the magic update we need to remove local dependencies.
+      - name: Npm clean install
```

Comment:
I might just call this "fix package locks" cuz that's the important thing it's doing! The actual command being run doesn't really matter. But this is an unimportant change so feel free to ignore :P